tests/kubectl_test.go

```diff
@@ -74,6 +74,6 @@ var _ = Describe("[rfe_id:3423][vendor:cnv-qe@redhat.com][level:component]oc/kub
 		Expect(resultFields[len(resultFields)-verifyPos]).To(Equal(expectedData))
 	},
 		table.Entry("[test_id:3421]virtualmachine", "get", "vm", "wide", []string{"NAME", "AGE", "RUNNING", "VOLUME", "CREATED"}, 1, "true"),
-		table.Entry("[test_id:3422]virtualmachineinstance", "get", "vmi", "wide", []string{"NAME", "AGE", "PHASE", "IP", "NODENAME", "LIVE-MIGRATABLE"}, 1, "True"),
+		table.Entry("[test_id:3422]virtualmachineinstance", "get", "vmi", "wide", []string{"NAME", "AGE", "PHASE", "IP", "NODENAME", "LIVE-MIGRATABLE"}, 1, "False"),
```

Comment:
I can't remember whether I've seen this last time, but it does look like the `expectedData` is comparing case sensitive? So just out of interest: Does the difference matter? I suppose it does? So therefore the test data in the table looks odd in using once `true` (lowercase) and once `False` (uppercase)?

Reply:
I relate to this finding. I only updated this test to make it pass.
The returning values are indeed inconsistent and it would deserve the fix.

This would be nice followup PR.

Reply:
Ack. Would be great if you created an issue for it so we don't forget.

Reply:
I created issue #3072 it will make a great starting task.